common/ideas/sweden.txt

```diff
@@ -344,7 +344,7 @@ ideas = {
 			}
 			
 			modifier = {
-				offence = 0.1
```

Comment:
offence is correct, it is a separate modifier from army_attack_factor and affects air/naval attack as well. Same goes below for defence.

Reply:
I've been looking into this and it seems that whilst offence/defence should work, they don't. Not for land combat at least, I don't know about air/naval combat. I increased the offence/defence value significantly to test but it make no difference to anything visible on the template, division or battle views. So either it used to work and no longer does or it's has an effect which isn't displayed anywhere.

Are you certain it works and hasn't been deprecated?

I'd suggest keeping my change but additionally adding the corresponding modifiers for air and naval combat. This would be helpful too as I had no idea those focuses are intended to buff your air force and navy as well as land forces. (Also, it could be cool to add a couple of focuses to choose between offence and defence for the air force and navy separately.)